src/main/java/gregtech/api/items/GT_MetaBase_Item.java

```diff
@@ -143,16 +145,19 @@ public EnumActionResult onItemUse(ItemStack aStack, EntityPlayer aPlayer, World
         use(aStack, 0, aPlayer);
         isItemStackUsable(aStack);
         ArrayList<IItemBehaviour<GT_MetaBase_Item>> tList = mItemBehaviors.get((short) getDamage(aStack));
-        if (tList != null) for (IItemBehaviour<GT_MetaBase_Item> tBehavior : tList)
-            if (tBehavior.onItemUse(this, aStack, aPlayer, aWorld, blockPos, aSide, hitX, hitY, hitZ, hand)) {
-                if (aStack.stackSize <= 0) aPlayer.setHeldItem(hand, null);
-                return EnumActionResult.SUCCESS;
+        if (tList != null) {
+            for (IItemBehaviour<GT_MetaBase_Item> tBehavior : tList) {
+                if (tBehavior.onItemUse(this, aStack, aPlayer, aWorld, blockPos, aSide, hitX, hitY, hitZ, hand)) {
+                    if (aStack.stackSize <= 0) aPlayer.setHeldItem(hand, null);
+                    return EnumActionResult.SUCCESS;
+                }
             }
+        }
         if (aStack.stackSize <= 0) {
             aPlayer.setHeldItem(hand, null);
             return EnumActionResult.SUCCESS;
         }
-        return EnumActionResult.SUCCESS;
```

Comment:
This change looks like it will fix https://github.com/jaquadro/StorageDrawers/issues/369